po/R-es.po

```diff
@@ -168,27 +168,27 @@ msgstr "Tutoriales"
 #: development.R:20
 #, fuzzy
 msgid "Released version"
-msgstr "Version official"
+msgstr "Versión lanzada"
 
 #: development.R:21
 msgid "In-development version"
 msgstr "Versión en desarrollo"
 
 #: development.R:22
 msgid "Unreleased version"
-msgstr "Versión no en CRAN"
+msgstr "Versión en desarrollo"
 
 #: navbar.R:97
 msgid "Reference"
 msgstr "Ayuda"
 
```

Comment:
Si esta sería la traducción para la lista de funciones, no sé si con "ayuda" quede tan claro. Podría ser "Referencia"

Reply:
For more context, this is the link in the navbar that takes you to the index that lists all `.Rd` files. I don't think "reference" is particularly clear in English, either, but it's the best we could come up with.

Reply:
My concern about "Ayuda" is that people may think they will find help there. @dieghernan, what do you think of something like "guía", "guía de referencia", "índice"?

Reply:
I like índice, as in a book

Reply:
Finalmente: Índice